### DISCUSSION TITLE
E-mail notifications

### DISCUSSION THREAD

**Discussion by smGoblin** [requarks/wiki#6518] [Help / Questions] [answered]
Hello, colleagues! Can you please tell me if there are any possibilities to use notification by e-mail? Mailing lists or something like that?
Thank you!

**Comment by prodigy7** [comment]
For me, a possibility to send notifications according to defined criteria would be great. For example, there may be important articles where you want to make sure that everyone takes note that there have been changes (see https://github.com/requarks/wiki/discussions/5614). This would then fit in with the release feature, for example, that when a new version is released, the notification is sent.

**Comment by NGPixel** [comment, marked as answer]
Follow https://feedback.js.wiki/wiki/p/notifications-email-on-select-events
